proxy/src/auth/backend/jwt.rs

```diff
@@ -357,6 +358,20 @@ impl JwkCache {
     }
 }
 
+impl Default for JwkCache {
+    fn default() -> Self {
+        let client = Client::builder()
+            .user_agent(JWKS_USER_AGENT)
```

Comment:
this is not related to fix but I think it's a good habit to set `User-Agent` that allows identifying requests originating from Neon